modules/http-api/src/main/java/org/mule/runtime/http/api/server/AcceptsAllMethodsRequestMatcher.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+package org.mule.runtime.http.api.server;
+
+import static java.lang.System.identityHashCode;
+import static java.util.Collections.singletonList;
+import org.mule.runtime.http.api.domain.message.request.HttpRequest;
+
+/**
+ * Accepts all the HTTP methods.
+ */
+class AcceptsAllMethodsRequestMatcher extends DefaultMethodRequestMatcher {
```

Comment:
aren't all this matchers already present on the other PR?

Reply:
They are deleted on the other PR.